### PR TITLE
OSDOCS#8882 OLM 1.0 admonition for multicatalog conflicts

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OSDOCS-8882](https://issues.redhat.com//browse/OSDOCS-8882)

Link to docs preview:
* Components and arch > [About catalogs in OLM 1.0](https://71095--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/arch/olmv1-catalogd#olmv1-about-catalogs_olmv1-catalogd)
* Installing an Operator from a catalog > [About catalogs in OLM 1.0](https://71095--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/olmv1-installing-an-operator-from-a-catalog#olmv1-about-catalogs_olmv1-installing-operator)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Xia-Zhao-rh** [comment]
LGTM

**Review by m1kola — APPROVED** [review]
/lgtm

File: snippets/olmv1-multi-catalog-admon.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// * modules/olmv1-about-catalogs.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+If you try to install an Operator or extension that does not have unique name, the installation might fail or lead to an unpredictable result. This occurs for the following reasons:
+
+* If mulitple catalogs are installed on a cluster, {olmv1} does not include a mechanism to specify a catalog when you install an Operator or extension.
+* Dependency resolution in {olmv1-first} assumes that all of the Operators and extensions that are available to install on a cluster use a unique name for their bundles and packages.
```
Comment (mburke5678) [inline snippets/olmv1-multi-catalog-admon.adoc:12]: _assumes_ seems anthropomorphic to me. ISG identifies: _decide_, _think_, _want_ as anthropomorphic.

```suggestion
* Dependency resolution in {olmv1-first} requires/ensures that all of the Operators and extensions that are available to install on a cluster use a unique name for their bundles and packages.
```

**Comment by mburke5678** [comment]
@michaelryanpeter A minor nit. Otherwise LGTM

**Pull request merged**